src/pages/SingleTopicPage.jsx

```diff
@@ -2,20 +2,26 @@ import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 import { getTopicsByTopicType } from "../utils/fetch";
 import SingleArticle from "../components/SingleArticle";
+import Error from "../components/Error";
 
 const SingleTopicPage = () => {
   const [articles, setArticles] = useState([]);
   const [isLoading, setIsLoading] = useState(true)
+  const [error, setError] = useState();
   const { topictype } = useParams();
 
   useEffect(() => {
     getTopicsByTopicType(topictype).then((response) => {
       setArticles(response.data.articles);
       setIsLoading(false)
+    }).catch((err) => {
+        setError({msg: err.response.data.msg, status: err.response.status})
+        setIsLoading(false)
     });
```

Comment:
This may or may not be a backend issue, but currently if an invalid article id is provided the page crashes
so for example `/articles/katherine`
Have a look at what the error looks like and why this might be happening 🔍